### ISSUE TITLE
Tune K

### ISSUE THREAD

**Issue by jackvfb** [jackvfb/south-pole-ozone-hole#3]
I think rather than make a branch for different arbitrary choices of K, I should so I more formal train/test split and validation to optimize K.

**Comment by jackvfb** [comment]
I can holdout some years, say the last three years and do kfold cross validation on the training data, then get metrics from the test set.
